python-builder/bin/build

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+set -euo pipefail
+
+# build source & binary distributions (wheel)
+pipenv run python setup.py sdist bdist_wheel
```

Comment:
A few things on this script:
1. It has to place the built artifact in the `/code/dist` folder. 
2. It has to output the name of that artifact.
3. It cannot output anything other than that name.

Reply:
1. setup.py automatically places the artifact in the dist/ folder, so no change needed there.
2. Made the change so the .tar.gz source archive is echo'ed.